arangod/Aql/CalculationExecutor.cpp

```diff
@@ -147,6 +147,17 @@ bool CalculationExecutor<calculationType>::shouldExitContextBetweenBlocks() cons
   return isRunningInCluster || stream;
 }
 
+template <>
+void CalculationExecutor<CalculationType::Constant>::doEvaluation(InputAqlItemRow& input,
+                                                                   OutputAqlItemRow& output) {
+  TRI_IF_FAILURE("CalculationBlock::executeExpression") {
+    THROW_ARANGO_EXCEPTION(TRI_ERROR_DEBUG);
+  }
+
+  // TODO - really nothing to do here?
+  // we at least have to set _inputRowCopied, otherwise subsequent assertions fail
```

Comment:
We should clarify whether this is necessary before merging this PR.

Reply:
This should be obsolete once we completely remove the CalculationNode from the plan.